### PR TITLE
For now adding a stub, more research is required

### PR DESCRIPTION
**Pull request by sappigeninja**
For now im adding a stub for the FIONREAD operation.
Im doing some more research on how to implement it (if its actually needed that is).

### CODE REVIEW

File: sysdeps/managarm/generic/file.cpp [hunk 1]
```diff
@@ -1986,7 +1986,7 @@ int sys_ioctl(int fd, unsigned long request, void *arg, int *result) {
 		auto send_req = parseSimple(element);
 		auto recv_resp = parseInline(element);
 		auto recv_list = parseLength(element);
-
+nn
```
Comment (Dennisbonke) [inline sysdeps/managarm/generic/file.cpp:1989]: I assume this is committed by accident?

Comment (sappigeninja) [reply sysdeps/managarm/generic/file.cpp]: jup
fixed it just now

**Comment by avdgrinten** [comment]
We silently return success here. That is probably not a good idea. What do we need this ioctl for? We should probably implement it properly.

**Comment by qookei** [comment]
Superseded by #358

**Pull request closed without merging**